VirtoCommerce.CustomerModule.Data/Search/Indexing/MemberDocumentBuilder.cs

```diff
@@ -162,5 +166,52 @@ protected virtual void AddParentOrganizations(IndexDocument document, ICollectio
             document.AddFilterableValues("ParentOrganizations", nonEmptyValues);
             document.AddFilterableValue("HasParentOrganizations", nonEmptyValues?.Any() ?? false);
         }
+
+        protected virtual void IndexDynamicProperties(IndexDocument document, ICollection<DynamicObjectProperty> properties)
+        {
+            foreach (var property in properties)
```

Comment:
Can you move this loop into caller side, and define this function only for single DynamicObjectProperty parameter